.github/workflows/tests-karma-editor.yml

```diff
@@ -101,9 +101,9 @@ jobs:
 
       - name: Run integration tests with Percy
         run: |
-          npx percy exec --parallel --config=percy.config.karma.yml -- \
+          npx percy exec --quiet --parallel --config=percy.config.karma.yml -- \
             npm run test:karma:story-editor -- --headless --viewport=1600:1000 --coverage --shard=$SHARD || \
-          npx percy exec --parallel --config=percy.config.karma.yml -- \
+          npx percy exec --quiet --parallel --config=percy.config.karma.yml -- \
```

Comment:
👍

Reply:
Now it will just print errors and skip:
```
2022-09-19T08:21:26.5310152Z .[percy] Snapshot taken: Inline style override should keep formatting when all text is replaced: "A new text" in bold
2022-09-19T08:21:33.1766490Z .[percy] Snapshot taken: Inline style override should have correct formatting deleting text with one formatting, ending up in different formatting: "Full in some text" in bold
2022-09-19T08:21:36.8470745Z .[percy] Snapshot taken: Inline style override CUJ: Creator Can Style Text: Apply B, Apply I should have correct formatting when already italic, then inline removing italic and adding bold, then inserting text: "Fifooll in some text" in mixed formatting
2022-09-19T08:21:44.4785788Z .[percy] Snapshot taken: Inline style override when cursor is placed after second character CUJ: Creator Can Style Text: Apply Uppercase should have correct formatting when pressing uppercase toggle, then inserting text: "Fifooll in some text" in mixed formatting
2022-09-19T08:21:49.4178276Z .[percy] Snapshot taken: Inline style override when cursor is placed after second character CUJ: Creator Can Style Text: Select weight should have correct formatting when selecting font weight, then inserting text: "Fifooll in some text" in mixed formatting
2022-09-19T08:21:53.0389335Z .[percy] Snapshot taken: Inline style override when cursor is placed after second character CUJ: Creator Can Style Text: Apply B should have correct formatting when pressing bold toggle, then inserting text: "Fifooll in some text" in mixed formatting
2022-09-19T08:21:55.9117637Z .[percy] Snapshot taken: Inline style override when cursor is placed after second character CUJ: Creator Can Style Text: Apply B should have correct formatting when pressing mod+b for bold, then inserting text: "Fifooll in some text" in mixed formatting
2022-09-19T08:21:58.7617807Z .[percy] Snapshot taken: Inline style override when cursor is placed after second character CUJ: Creator Can Style Text: Apply U, Apply I should have correct formatting when pressing underline toggle, then mod+i, then inserting text: "Fifooll in some text" in mixed formatting
2022-09-19T08:22:01.4889864Z .[percy] Snapshot taken: Inline style override CUJ: Creator can Add and Write Text: Select all text inside a textbox, Write/edit text should keep formatting when all text is removed, then replaced: "Even more text" in bold
```